includes/class-wc-payments-account.php

```diff
@@ -2049,6 +2049,13 @@ private function init_stripe_onboarding( string $setup_mode, string $wcpay_conne
 			$gateway->update_option( 'enabled', 'yes' );
 			$gateway->update_option( 'test_mode', empty( $onboarding_data['is_live'] ) ? 'yes' : 'no' );
 
+			$capabilities = $this->onboarding_service->get_capabilities_from_request();
```

Comment:
Please add more inline docs here explaining where these capabilities come from and why we enforce them.

Reply:
Included an explanation with inline documentation.